libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
If __ICCARM__ is undefined, it will be ARMCC or GCC ARM.
Since ARMCC and GCC ARM have a return value in "__disable_irq()", "was_masked" is valid value.